SMLHelper/Patchers/OptionsPanelPatcher.cs

```diff
@@ -94,5 +110,167 @@ internal static void AddTabs_Postfix(uGUI_OptionsPanel __instance)
                 }
             }
         }
+
+
+
+        // do not show tab if it is already visible (to prevent scroll position resetting)
+        internal static bool SetVisibleTab_Prefix(uGUI_TabbedControlsPanel __instance, int tabIndex)
+        {
+            return !(tabIndex >= 0 && tabIndex < __instance.tabs.Count && __instance.tabs[tabIndex].pane.activeSelf);
```

Comment:
Readability: Resolve the boolean statement by propagating the negation.
```CS
return tabIndex < 0 || tabIndex >= __instance.tabs.Count || !__instance.tabs[tabIndex].pane.activeSelf;
```
References: https://en.wikipedia.org/wiki/Negation#Distributivity